projects/extension/ai/vectorizer.py

```diff
@@ -32,6 +32,12 @@ def execute_vectorizer(plpy, vectorizer_id: int) -> None:
     embedding_api_key = (
         vectorizer.get("config", {}).get("embedding", {}).get("api_key_name", None)
     )
+    destination = vectorizer.get("config", {}).get("destination", None)
+    if destination is not None:
+        vectorizer["target_schema"] = destination.get("target_schema", vectorizer.get("source_schema"))
+        vectorizer["target_table"] = destination.get("target_table", vectorizer.get("source_table"))
+        vectorizer["view_schema"] = destination.get("view_schema", vectorizer.get("source_schema"))
+        vectorizer["view_table"] = destination.get("view_table", vectorizer.get("source_table"))
```

Comment:
Why the fallback to `vectorizer.get`? If destination is not `None`, then we wouldn't expect those values to be populated?

Reply:
yeh true makes no sense, null would break the receiving end anyways since not empty is asserted.